Utilizr.Vpn.Ras/RasNativeProvidor.cs

```diff
@@ -248,6 +314,8 @@ void RasmanRunning()
             const string service = "rasman";
             try
             {
+                ChangeServiceStartType(service, ServiceStartupType.Manual);

```

Comment:
I think we need to do two things here.

1) Wrap in a try/catch so if we failed to do this for whatever reason, we don't prevent the connection attempt. Having this service disabled should be an edge case scenario, rather than the majority.

2) I think we should only be attempting to change the startup type only if it's `Disabled`. Otherwise we're potentially changing a lot of user's settings.

It looks like there is a [QueryServiceConfig](https://learn.microsoft.com/en-us/windows/win32/api/winsvc/nf-winsvc-queryserviceconfigw) function to get a [QueryServiceConfig](https://learn.microsoft.com/en-us/windows/win32/api/winsvc/ns-winsvc-query_service_configw) struct which will include the startup type.